vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,7 +1,139 @@
-# Entando 7.x Release Notes
+# Entando 7.1 Release Notes
 
-## New Features and Major Fixes
-This information will be provided when the next version of Entando is released.
+Entando 7.1.0 is an important feature release that introduces major changes to the bundle subsystem. It supports a brand new create phase, bundle service discovery and App Builder extensibility via Entando Packaged Capabilities (EPCs).
+
+## ABSTRACT
+
+- Security fixes
+- Node-based CLI module to manage bundle projects
+- Decoupled MFE ⇒ MS communication via service discovery ([API claims](../getting-started/ent-api.md))
+- New docker-based bundles (bundles entirely distributed via Docker/OCI images)
+- App Builder is now extensible through bundles (supports override of core App Builder components and the addition of EPC extensions)
+- App Builder main menu is converted to a overridable bundle containing an MFE and BFF service
+- Widgets, pages and bundle assets are now created within the scope of a specific bundle
+- Performance optimizations
+
+### Compatibility
+
+* [Entando 7.1 Compatibility](https://entando.com/entando-de-app/cmsresources/cms/documents/Entando_7.1_Compatibility.pdf)
+* [Guides for previous versions of Entando](https://entando.com/page/en/compatibility-guide)
+
+## BUG FIXES AND NEW FEATURES
+
+### Security Updates
+
+#### Dependencies
+
+- Fixed CVE-2022-22969 (spring-security, DoS)
+- Fixed CVE-2021-31805 (struts2, RCE)
+- Fixed CVE-2022-22965 (spring-beans, RCE)
+- Fixed CVE-2022-22950 (spring-expression, DoS)
+- Fixed CVE-2022-22968 (spring-context)
+- Fixed CVE-2022-26336 (apache poi-scratchpad, DoS)
+- Fixed CVE-2022-30126 (apache tika, DoS)
+- Fixed CVE-2022-30973 (apache tika, DoS)
+- Fixed CVE-2022-22976 (spring-security)
+- Fixed CVE-2022-22978 (spring-security-web, Authorization Bypass)
+- Fixed CVE-2022-33879 (apache tika, DoS)
+- Fixed CVE-2022-24823 (apache netty, Information Exposure)
+
+### **AdminConsole:**
```

Comment:
is there a reason why this is bolded with a colon?

Reply:
yes lol i wanted to confirm it should be "Admin Console" but missed it during our triage

Reply:
Lose this section, combine with App Builder.